packages/react-core/src/components/Select/examples/Select.md

```diff
@@ -1964,403 +2126,130 @@ class MultiTypeaheadSelectInputWithChipGroupProps extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      isOpen: false,
-      isDisabled: false,
-      direction: SelectDirection.down
-    };
+### Multiple typeahead with custom objects
 
-    this.onToggle = isOpen => {
-      this.setState({
-        isOpen
-      });
-    };
-
-    this.toggleDisabled = checked => {
-      this.setState({
-        isDisabled: checked
-      });
-    };
-
-    this.toggleDirection = () => {
-      if (this.state.direction === SelectDirection.up) {
-        this.setState({
-          direction: SelectDirection.down
-        });
-      } else {
-        this.setState({
-          direction: SelectDirection.up
-        });
-      }
-    };
-  }
-
-  render() {
-    const { isOpen, selected, isDisabled, direction } = this.state;
-    const titleId = 'title-id-2';
-    return (
-      <div>
-        <span id={titleId} hidden>
-          Title
-        </span>
-        <Select
-          aria-label="Select Input"
-          onToggle={this.onToggle}
-          isOpen={isOpen}
-          aria-labelledby={titleId}
-          isDisabled={isDisabled}
-          direction={direction}
-          customContent="[Panel contents here]"
-          placeholderText="Filter by birth month"
-        />
-        <Checkbox
-          label="isDisabled"
-          isChecked={this.state.isDisabled}
-          onChange={this.toggleDisabled}
-          aria-label="disabled checkbox panel"
-          id="toggle-disabled-panel"
-          name="toggle-disabled-panel"
-        />
-        <Checkbox
-          label="Expands up"
-          isChecked={direction === SelectDirection.up}
-          onChange={this.toggleDirection}
-          aria-label="direction checkbox panel"
-          id="toggle-direction-panel"
-          name="toggle-direction-panel"
-        />
-      </div>
-    );
-  }
-}
-```
-
-### Appending document body vs parent
-
-Avoid passing in `document.body` when passing a value to the `menuAppendTo` prop on the Select component, as it can cause accessibility issues. These issues can include, but are not limited to, being unable to enter the contents of the Select options via assistive technologies (like keyboards or screen readers).
-
-Instead append to `"parent"` to achieve the same result without sacrificing accessibility.
-
-In this example, while, when the dropdown is opened, both Select variants handle focus management within their dropdown contents the same way, you'll notice a difference when you try pressing the Tab key after selecting an option.
-
-For the `document.body` variant, the focus will be placed at the end of the page, since that is where the dropdown content is appended to in the DOM (rather than focus being placed on the second Select variant as one might expect). For the `"parent"` variant, however, the focus will be placed on the next tab-able element (the "Toggle JS code" button for the code editor in this case).
+A `<SelectOption>` may be a custom object.
```

Comment:
To borrow some verbiage from the description of the PR where this was introduced (https://github.com/patternfly/patternfly-react/pull/2612):

```suggestion
A `<SelectOption>` can have an object passed into the `value` property in order to store additional data beyond just a string value. The object passed in must have a `toString` function that returns a string to display in the `SelectMenu`.
```

Or something along those lines.